source/product-overview/mattermost-v10-changelog.md

```diff
@@ -4,6 +4,44 @@
 ```{include} common-esr-support-upgrade.md
 ```
 
+(release-v10.12-feature-release)=
+## Release v10.12 - [Feature Release](https://docs.mattermost.com/product-overview/release-policy.html#release-types)
+
+**Release day: 2025-09-16**
+
+```{Important}
+If you upgrade from a release earlier than v10.10, please read the other [Important Upgrade Notes](https://docs.mattermost.com/administration-guide/upgrade/important-upgrade-notes.html).
```

Comment:
Should this be v10.11?

Reply:
There were no important upgrade notes in v10.11.